File: atividade-3/005_final/005_final.js
//Defina a função final tal que final xs é uma lista formada pelos n elementos finais de xs.

const size = 2;
const list = [2, 5, 4, 7, 9, 6];

const final = (n, xs) => (n > 0 ? xs.slice(-n) : []);

console.log(final(size, list));

File: atividade-3/005_final/005_final.hs
module Main where

-- Defina a função final tal que final xs é uma lista formada pelos n elementos finais de xs.

final :: Int -> [Int] -> [Int]
final n xs = drop $length xs n xs

main = do
  a <- readLn :: IO Int
  b <- readLn :: IO [Int]
  print $ final a b